app/javascript/views/plans/new.js

```diff
@@ -37,6 +37,7 @@ $(() => {
         if (data.templates.length === 1) {
           $('#plan_template_id option').attr('selected', 'true');
           $('#multiple-templates').hide();
+          $('#available-templates').fadeOut();
```

Comment:
Some JS bugs on the Plan create page from the initial ROR work (if a funder was selected, then the 'no funder' box was ticked it was not removing the loaded templates)